boards/arm/96b_ivy5661/dbghdr.c

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2018, UNISOC Incorporated
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifdef BOARD_96B_IVY5661
+__attribute__ ((section(".dbghdr")))
+const unsigned long ulDebugHeader[] = {
```

Comment:
Is this a board specific header, or something we'd need for this SoC always?

Reply:
This is not used by the SoC, remove this file.